pkg/controller/serviceaccount/tokens_controller.go

```diff
@@ -189,14 +195,14 @@ func (e *TokensController) queueServiceAccountUpdateSync(oldObj interface{}, new
 }
 
 // complete optionally requeues key, then calls queue.Done(key)
-func (e *TokensController) retryOrForget(logger klog.Logger, queue workqueue.RateLimitingInterface, key interface{}, requeue bool) {
+func retryOrForget[T comparable](logger klog.Logger, queue workqueue.TypedRateLimitingInterface[T], key T, requeue bool, maxRetries int) {
```

Comment:
a pity that a method cannot have its own type parameter 🤷‍♂️

Reply:
Yeah, this drastically reduces the usefulness of generics